tasks/startSelenium.js

```diff
@@ -0,0 +1,84 @@
+var _ = require('lodash');
+var request = require('request');
+var fs = require('fs');
+var path = require('path');
+var colors = require('ansicolors');
+var crypto = require('crypto');
+var {spawn} = require('child_process');
+
+module.exports = function (grunt) {
+  grunt.registerTask('startSelenium', 'Start an instance of selenium standalone', function (keepalive) {
+    const done = this.async();
+    const config = _.defaults(this.options(), {
+      seleniumStandalone: {
+        filename: 'selenium-server-standalone-2.47.1.jar',
+        server: 'https://selenium-release.storage.googleapis.com/2.47/',
+        md5: 'e6cb10b8f0f353c6ca4a8f62fb5cb472'
+      }
+    });
+
+    const SELENIUM_DIRECTORY = path.join(grunt.config.get('root'), 'selenium');
+    const SELENIUM_FILE_PATH = path.join(SELENIUM_DIRECTORY, config.seleniumStandalone.filename);
+    const SELENIUM_DOWNLOAD_URL = config.seleniumStandalone.server + config.seleniumStandalone.filename;
+    const SELENIUM_HASH = config.seleniumStandalone.md5;
+
+    function waitForReadiness(seleniumProcess) {
+      seleniumProcess.stderr.on('data', function (log) {
```

Comment:
This will probably work most of the time, but you might be better of creating a `run:seleniumServer` task, since that's the purpose of grunt-run